interpreter/interpreter.py

```diff
@@ -192,6 +192,11 @@ def reset(self):
     Resets the interpreter.
     """
     self.messages = []
+    
+    #遍历并stop每一个code interpreter
+    for code_interpreter in self.code_interpreters.values():
+      code_interpreter.stop()
```

Comment:
I think this needs to move to [`interpreter/core/core.py`](https://github.com/KillianLucas/open-interpreter/blob/main/interpreter/core/core.py) now that the codebase has been refactored.